src/psyclone/domain/lfric/kernel/inter_grid_arg.py

```diff
@@ -58,6 +58,8 @@ class InterGridArg(FieldArg):
         InterGrid arg is on (coarse or fine).
 
     '''
+    mesh_arg_index = 4
```

Comment:
Please add a comment describing what this holds.

Reply:
Done.